src/common/styles/subHeadings.css

```diff
@@ -0,0 +1,6 @@
+.subHeadings {
+  color: var(--subHeading-color);
```

Comment:
how does it work without import?

Reply:
magic https://github.com/folio-org/ui-organizations/blob/master/src/Main/Main.css